ganttproject/src/main/java/net/sourceforge/ganttproject/GanttProject.java

```diff
@@ -161,6 +171,11 @@ public GanttProject(boolean isOnlyViewer) {
     ProjectStateHolderEventListener stateListener = new ProjectStateHolderEventListener(myProjectDatabase);
     addProjectEventListener(stateListener);
     getTaskManager().addTaskListener(stateListener);
+    if (isColloboqueLocalTest()) {
+      getWebSocket().register(null);
```

Comment:
is it really needed? It seems that it only starts sending heartbeats (which colloboque is ignoring anyway)

Reply:
The only thing it does is connecting to the web socket for the first time. We need to do it so that it's the only place where we specify the url, right? 
Btw, heartbeats are not sent as they are initiated in `onAuthDone`